exporter/opencensusexporter/opencensus.go

```diff
@@ -105,20 +105,19 @@ func OpenCensusTraceExportersFromViper(v *viper.Viper) (tes []exporter.TraceExpo
 	return tes, mes, doneFns, nil
 }
 
-func (oce *ocagentExporter) ExportSpans(ctx context.Context, td data.TraceData) error {
+func (oce *ocagentExporter) ExportSpans(ctx context.Context, td data.TraceData) (err error) {
+	observabilityRecorder := internal.NewExporterEventRecorder("ocagent")
+
+	ctx = observabilityRecorder.Start(ctx, td.Node)
+	defer observabilityRecorder.End(ctx, int64(len(td.Spans)), err)
```

Comment:
err here will always be nil I believe. https://play.golang.org/p/kFOauu3Y7YF Similar to above, probably need a closure here